src/PointwiseFunctions/Hydro/TransportVelocity.cpp

```diff
@@ -0,0 +1,42 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Hydro/TransportVelocity.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// \cond
+namespace hydro {
+template <typename DataType, size_t Dim, typename Fr>
+tnsr::I<DataType, Dim, Fr> transport_velocity(
```

Comment:
Could you add a `not_null` overload please? Otherwise this function will be too expensive to use.